docs/docs/accessibility.md

```diff
@@ -0,0 +1,320 @@
+---
+id: accessibility
+title: Accessibility
+permalink: docs/accessibility.html
+redirect_from:
+  - "docs/index.html"
+---
+
+## Why accessibility?
+
+Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. People who experience disabilities only fully benefit from websites when 
```

Comment:
Agreed. I was struggling with this myself when I wrote it. Trying to avoid 'othering' while coming up with something that would speak to the developer who so not know about a11y yet. This one won then but your signal is clear enough that the vote should have gone the other way. Thank you! Fixed in the next push.